### ISSUE TITLE
[MISC_8.3.1] Expedita vel sit.

### ISSUE THREAD

**Issue by smockle** [smockle/action-autolabel#1]
Eligendi rerum deleniti quia repudiandae velit at non cumque `class="js-button button"`. Ipsum tempore non doloribus aut labore autem fuga hic a nested `<div>` inside a `<span>`. Sit distinctio est occaecati pariatur ad aut similique et voluptas. Sapiente ut omnis enim quae libero voluptatem unde et.

#### Fails

[4.1.1 Parsing (level A)](https://www.w3.org/WAI/WCAG22/Understanding/parsing)

cc @smockle

View the [repository](https://github.com/smockle/action-autolabel).

**Comment by smockle** [comment]
ℹ️ This issue exists for testing the [`smockle/action-autolabel` action](https://github.com/smockle/action-autolabel)
